schemas/com.snowplowanalytics.mobile/remote_config/jsonschema/1-0-0

```diff
@@ -0,0 +1,160 @@
+{
+  "$schema" : "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self" : {
+    "vendor" : "com.snowplowanalytics.mobile",
+    "name" : "remote_config",
+    "version" : "1-0-0",
+    "format" : "jsonschema"
+  },
+  "description": "The configuration file for the Snowplow mobile trackers.",
+  "type": "object",
+  "properties": {
+    "configurationVersion": {
+      "description": "Version number that identifier the current configuration. It has to be increased on each update.",
```

Comment:
Presumably we can add `minimum`?

Reply:
Technically it wouldn't need a minimum as we use it just to check if the configuration is changed.
We can eventually put it to 0 but there aren't technical reasons for that.

Reply:
You wouldn't want it to be a negative number though, so specifying a minimum probably still makes sense rather than omitting it.

Reply:
Why all this anger against negative numbers :D